README.md

```diff
@@ -1,4 +1,4 @@
-# p2pmem-test
+# Introduction
```

Comment:
@amaier17 can we start with

"# p2pmem-test: A tool for userspace testing of p2pdma

## Introduction"

and then move forward from there?